tools/ptf/Dockerfile

```
@@ -7,8 +7,9 @@
 
 ARG GRPC_VER=1.26
 ARG PROTOBUF_VER=3.12
+ARG SCAPY_VER=2.4.3
 
-FROM python:2.7.13 as proto-deps
+FROM python:3.5 as proto-deps
```

Python 3.5 has reached end-of-life
```suggestion
FROM python:3.9 as proto-deps
```